README.md

```diff
@@ -38,6 +38,14 @@ Use `--help` option in order to get the full list of options
 --help        Show this message and exit.
 ```
 
+## Usage with srt-live-transmit
+As mentioned this tool is intended for usage with [srt-live-transmit](https://github.com/Haivision/srt/blob/master/docs/apps/srt-live-transmit.md), for which you explicitly need to enable the export of statistics in .csv format. This can be done by passing the following parameters:
```

Comment:
Not exactly, another test applications like srt-xtransmit (I refer to it a bit earlier) or any other application producing csv files with the same column names can be used as a generator of SRT statistics for the script. So let's formulate this in a bit different way.

"As mentioned this tool is intended for usage with [srt-live-transmit](https://github.com/Haivision/srt/blob/master/docs/apps/srt-live-transmit.md), for which you explicitly need to enable the export of statistics in .csv format." -> "For the usage with [srt-live-transmit](https://github.com/Haivision/srt/blob/master/docs/apps/srt-live-transmit.md), make sure that the export of SRT statistics is enabled in `.csv` format. This can be done ..."

Reply:
Resolved in https://github.com/koenkarsten/srt-stats-plotting/commit/f2c97e0c19c8efa5cc1b9bc5b4721879d260376a